### ISSUE TITLE
James check Jacknife stuff

### ISSUE THREAD

**Issue by meghapsimatrix** [meghapsimatrix/simhelpers#3]
Please 😸

**Comment by jepusto** [comment]
Looking this over. For the Welch simulations, I think it would be better to use unequal sample sizes so that the variance estimates for the regular t-test are different than for the Welch t-test. With equal sample sizes, they're identical, which makes the example of relative bias calculations less interesting.

**Comment by meghapsimatrix** [comment]
Thanks for the comment. I was thinking of doing that but been distracted. I just changed it so that there is equal size (50, 50) and unequal (50, 70) and it definitely shows in relative var and in type 1 error and conf intervals. I need to double check all the vignettes to make sure they are consistent with this change. Thanks a lot again! Good night!

**Comment by meghapsimatrix** [comment]
One comment. On the relative RMSE MCSE formula you wrote 1/ K times theta squared
![image](https://user-images.githubusercontent.com/33493716/77502897-2ebc5e00-6e2a-11ea-8d8d-14970140267b.png)

But that doesn't show up in the relative variance table
![image](https://user-images.githubusercontent.com/33493716/77502977-62978380-6e2a-11ea-886f-ea82401e7cda.png)

I think if you think that squared theta should be added to relative rmse it should be added to all the relative variance estimates, right? 

If I need to add it, I need to change all the formulas in the functions.

**Comment by jepusto** [comment]
This depends on how you're doing the jackknife. In the first case (where
the true parameter value is known), are you jackknifing the raw RMSE or the
relative RMSE? If raw, then you'll need to divide by the parameter. If
you're jackknifing the relative RMSE, then of course you don't need to
divide by the parameter again, but the formula in the table should be
clarified that the jackknifes are on relative RMSE (maybe abbreviate as
rRMSE?).

In the second case, it's the relative MSE and relative RMSE that need to be
jackknifed, so the formulas should probably clarify that (rMSE and rRMSE?).

**Comment by meghapsimatrix** [comment]
I jackknifed the relative RMSE. I changed the notations. Phew. I thought all the formulas were wrong 🤒 but all good!

![image](https://user-images.githubusercontent.com/33493716/77550010-93a1a380-6e7e-11ea-9898-9619006973cb.png)

![image](https://user-images.githubusercontent.com/33493716/77550042-9e5c3880-6e7e-11ea-9a9e-833f007779cf.png)

Also, I redid welch_res with equal vs unequal sample sizes.
